components/tools/OmeroJava/test/integration/AbstractServerTest.java

```diff
@@ -698,8 +698,8 @@ protected void makeGroupOwner() throws Exception {
      */
     protected omero.client disconnect() throws Exception {
         omero.client oldClient = client;
-        client = null;
         clean();
+        client = null;
```

Comment:
The ``disconnect`` method was never invoking the ``del`` method nothing related to ice 3.6